### DISCUSSION TITLE
@seahoh，最近有什么新招没？

### DISCUSSION THREAD

**Discussion by wpyok500** [SeaHOH/GotoX#61] [一般]
@seahoh，最近有什么新招没？
```

**Comment by SeaHOH** [comment]
有想法，没动作。
ipv6 和 sni 都还好使，quic 也有几个能用的，不是很迫切；主要还是没太多空闲时间。

**Comment by wpyok500** [comment]
还是这三板斧，我一直用quic，服务器端有打算更新没？

**Comment by SeaHOH** [comment]
GAE 基本上没啥可更新的了，除非又要兼容其它客户端。
其它云有时间会做，毕竟一个篮子不保险，不过要等挺久。

**Comment by wpyok500** [comment]
支持，支持，支持。
